pkg/projectfile/projectfile.go

```diff
@@ -1286,6 +1286,17 @@ func storeProjectMapping(cfg ConfigGetter, namespace, projectPath string) {
 
 			projectPath = filepath.Clean(projectPath)
 
+			for name, paths := range projects {
+				if name == namespace {
+					continue
+				}
+				for i, path := range paths {
+					if path == projectPath {
```

Comment:
We should pass these through `fileutils.ResolveUniquePath` (`path` here, `projectPath` we can replace the `filepath.Clean` on line 1287.